rom/laghos.cpp

```diff
@@ -658,9 +667,10 @@ int main(int argc, char *argv[])
 
     std::string offlineParam_outputPath = outputPath + "/offline_param" + romOptions.basisIdentifier + ".csv";
     romOptions.offsetType = getOffsetStyle(offsetType);
+    romOptions.indicatorType = getlocalROMIndicator(indicatorType);
     if (rom_online)
     {
-        std::string filename = solution_outputPath + "/ROMsol/romS_1";
+        std::string filename = testing_parameter_outputPath + "/ROMsol/romS_1";
```

Comment:
I realize `romS_1` was used before this PR, but I'm not sure why we don't check for `romS_0` in case there is only one window.

Reply:
I think the existence of `romS_0` is legit in the hyperreduce time marching phase since it is produced in the preparation phase. Anything beyond 0 is prohibited and 1 is used.